client/auth.js

```diff
@@ -11,8 +10,13 @@ export const authOptions = {
       clientSecret: process.env.GOOGLE_CLIENT_SECRET,
       async profile(profile) {
         // action to server
-        const res = await fetch(serverUrl + "user")
-        const {role} = await res.json()
+        const res = await fetch(serverUrl + "user", {
+          headers: {
+            "x-user-email": profile.email,
```

Comment:
This is okay for now, but it makes more sense to include the user's google id in their session, and send that to the backend to check roles, fetch forms, etc., instead of their email. Users can change the email associated with their account, but their ID is constant.

Reply:
Good point! Will add.